integration/docker/README.md

```diff
@@ -24,3 +26,38 @@ docker run -e ALLUXIO_MASTER_HOSTNAME=ec2-203-0-113-25.compute-1.amazonaws.com a
 Additional configuration files can be included when building the image by adding them to the
 `integration/docker/conf/` directory. All contents of this directory will be
 copied to `/opt/alluxio/conf`.
+
+
+## Building docker image with fuse support
+To build a docker image with fuse support, run
+
+```bash
+docker build -f Dockerfile.fuse -t alluxio-fuse .
+```
+
+You can add the same arguments supported by the non-fuse docker file.
+
+
+## Running docker image with fuse support
+There are a couple extra arguments required to run the docker image with fuse support, for example:
```

Comment:
could you also add the security implications of the extra parameter `SYS_ADMIN` or give a link?

Reply:
Done.